packages/data-point/lib/entity-types/base-entity/resolve.js

```diff
@@ -114,6 +115,24 @@ function createCurrentAccumulator (accumulator, reducer, entity) {
 
 module.exports.createCurrentAccumulator = createCurrentAccumulator
 
+/**
+ * Incase there is an override, assigns parameters to the correct requests.
```

Comment:
the description shouldn't mention requests, because the function is for all entity types